pallets/bootstrap/src/lib.rs

```diff
@@ -231,35 +234,35 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		/// provisions vested/locked tokens into the boostrstrap
-		#[pallet::weight(<<T as Config>::WeightInfo>::provision_vested())]
-		#[transactional]
-		pub fn provision_vested(
-			origin: OriginFor<T>,
-			token_id: TokenId,
-			amount: Balance,
-		) -> DispatchResult {
-			let sender = ensure_signed(origin)?;
-			let (vesting_starting_block, vesting_ending_block_as_balance) =
-				<<T as Config>::VestingProvider>::unlock_tokens(
-					&sender,
-					token_id.into(),
-					amount.into(),
-				)
-				.map_err(|_| Error::<T>::NotEnoughVestedAssets)?;
-			Self::do_provision(
-				&sender,
-				token_id,
-				amount,
-				ProvisionKind::Vested(
-					vesting_starting_block.saturated_into::<BlockNrAsBalance>(),
-					vesting_ending_block_as_balance.into(),
-				),
-			)?;
-			ProvisionAccounts::<T>::insert(&sender, ());
-			Self::deposit_event(Event::Provisioned(token_id, amount));
-			Ok(().into())
-		}
+		// /// provisions vested/locked tokens into the boostrstrap
+		// #[pallet::weight(<<T as Config>::WeightInfo>::provision_vested())]
+		// #[transactional]
+		// pub fn provision_vested(
+		// 	origin: OriginFor<T>,
+		// 	token_id: TokenId,
+		// 	amount: Balance,
+		// ) -> DispatchResult {
+		// 	let sender = ensure_signed(origin)?;
+		// 	let (vesting_starting_block, vesting_ending_block_as_balance) =
+		// 		<<T as Config>::VestingProvider>::unlock_tokens(
+		// 			&sender,
+		// 			token_id.into(),
+		// 			amount.into(),
+		// 		)
+		// 		.map_err(|_| Error::<T>::NotEnoughVestedAssets)?;
+		// 	Self::do_provision(
+		// 		&sender,
+		// 		token_id,
+		// 		amount,
+		// 		ProvisionKind::Vested(
+		// 			vesting_starting_block.saturated_into::<BlockNrAsBalance>(),
+		// 			vesting_ending_block_as_balance.into(),
+		// 		),
+		// 	)?;
+		// 	ProvisionAccounts::<T>::insert(&sender, ());
+		// 	Self::deposit_event(Event::Provisioned(token_id, amount));
+		// 	Ok(().into())
+		// }
```

Comment:
```suggestion
```
remove comments

Reply:
Well, the ask was to "disable the option to..." not "remove the feature..." so I just commented it.

Reply:
Please remove the code as @mateuszaaa suggested. We can always find this in the git history if needed.